coursebuilder/modules/explorer/templates/explorer.html

```diff
@@ -67,18 +67,17 @@
 
             <div class="col l4 m12 footer-social-icons">
                 <br>
-                <a href="https://plus.google.com/share?url=https://courses.academy.africa" target="_blank"
+                <a target="_blank"
```

Comment:
@Sharonsyra Missing hrefs?

Reply:
do we have these pages? Or i just have a dead link as a place holder?

Reply:
@Sharonsyra dead link until content is provided.